package.json

```diff
@@ -43,7 +43,7 @@
     "mini-css-extract-plugin": "^0.4.0",
     "optimize-css-assets-webpack-plugin": "^5.0.3",
     "typescript": "^3.6.4",
-    "uglifyjs-webpack-plugin": "^2.2.0",
+    "uglifyjs-webpack-plugin": "1.0.0-rc.0",
```

Comment:
Please add a coment here pointing to this PR and telling that the `v2.2.0` was downgraded to `v1.0.0-rc.0` because v2 breaks assets prod deploy when running: `npm run deploy`.
The idea is to address this in the future, trying to keep all deps at its latests versions.
Thanks!

Reply:
I certainly won't be able to comment a `json` file :grimacing:. What I can do is add a new attribute like `_comments` in the root of the file and add data there, what do you think?

Reply:
No problem, let it as it is! I've created a new issue for this: #33.
Will merge it, thanks again @sgobotta!